client/my-sites/checkout/composite-checkout/use-get-thank-you-url.js

```diff
@@ -230,19 +230,15 @@ function getFallbackDestination( {
 	return '/';
 }
 
-function maybeShowPlanBumpOfferConcierge( {
+function maybeShowPlanBumpOffer( {
 	pendingOrReceiptId,
-	orderId,
 	cart,
 	siteSlug,
 	didPurchaseFail,
 	isTransactionResultEmpty,
 } ) {
 	if ( hasPremiumPlan( cart ) && ! isTransactionResultEmpty && ! didPurchaseFail ) {
```

Comment:
This condition prevents the offer from being shown if the payment type is PayPal, or any of the other redirect payment types like Giropay.

> && ! isTransactionResultEmpty && ! didPurchaseFail

We needed this condition to be added to prevent users who cancel or fail their transactions being assigned to the test, but now that it's being launched to 100% users, we can safely remove this condition. So it will look like:

> if ( hasPremiumPlan( cart ) ) {